File: C/Exercicio03.c
//Faca um algoritimo que peca dois numeros e imprima a soma

#include<stdio.h>

int main(){
	
	int n1, n2, soma;
	
	printf("Informe o primeiro numero: ");
	scanf("%d", &n1);
	
	printf("Informe o segundo numero: ");
	scanf("%d", &n2);
	
	soma = n1 + n2;
	
	printf("A soma de %d + %d = %d", n1, n2, soma);
}

File: Java/Exercicio03.java
//Faça um algoritimo que peça dois numeros e imprima a soma

package Java;

import java.util.Scanner;

public class Exercicio03{
	
	public static void main(String[] args) {
		
		System.out.println("Informe o primeiro numero:");
		int n1 = new Scanner(System.in).nextInt();
		System.out.println("Informe o segundo numero:");
		int n2 = new Scanner(System.in).nextInt();
		
		int soma = n1 + n2;
		
		System.out.println("A soma de " + n1 + " + " + n2 + " = " + soma);
	}
}